DVR/HTTPURLResponse.swift

```diff
@@ -1,18 +1,18 @@
 import Foundation
 
-// There isn't a mutable NSHTTPURLResponse, so we have to make our own.
-class URLHTTPResponse: NSHTTPURLResponse {
+// There isn't a mutable HTTPURLResponse, so we have to make our own.
+class HTTPURLResponse: Foundation.HTTPURLResponse {
```

Comment:
If this code is not brought in via a pod (directly added to a project) which flavor of HTTPURLResponse will be used through the code base, Foundation or this? I would argue that this class should be prefixed.